tests/test_templatetags.py

```diff
@@ -0,0 +1,92 @@
+# coding: utf-8
+import functools
+
+import pytest
+from django.utils import translation
+from django_prices.templatetags import prices, prices_i18n
+
+from prices import Money, TaxedMoney, percentage_discount
+
+
+@pytest.fixture(scope="module")
+def money_fixture():
+    return Money("10", "USD")
+
+
+def test_templatetag_discount_amount_for():
+    price = TaxedMoney(Money(30, "BTC"), Money(30, "BTC"))
+
+    discount = functools.partial(percentage_discount, percentage=50)
+    discount_amount = prices.discount_amount_for(discount, price)
+    assert discount_amount == TaxedMoney(Money(-15, "BTC"), Money(-15, "BTC"))
+
+
+def test_non_existing_locale(money_fixture):
+    # Test detecting an error that occur for language 'zh_CN' for which
+    # the canonical code is 'zh_Hans_CN', see:
+    #     Babel 1.0+ doesn't support `zh_CN`
+    #     https://github.com/python-babel/babel/issues/37
+    # Though to make this test more reliable we mock the language with totally
+    # made up code 'oO_Oo' as the 'zh_CN' "alias" might work in the future, see:
+    #     Babel needs to support Fuzzy Locales
+    #     https://github.com/python-babel/babel/issues/30
+    translation.activate("oO_Oo")
```

Comment:
Please use `zz` for the nonexistent locale, you don't need to specify a variant.